### PR TITLE
Update etcd and Sensu version numbers in version API doc

### LINKED ISSUE #2665: Update /version API
### Description
Documentation and examples on the `/version` API needs to be updated with new etcd and sensu versions.

### Category

This is an issue with:

- [ ] errata (error in doc content)
- [ ] new doc (requires a completely new doc)
- [x] update (add missing or refresh existing content)
- [ ] improvement (improve docs content or format)

### Affected pages
https://docs.sensu.io/sensu-go/latest/api/version/

### Expected behavior
```
{
  "etcd": {
    "etcdserver": "3.3.22",
    "etcdcluster": "3.3.0"
  },
  "sensu_backend": "6.x.x"
}
```

### Current behavior
```
{
  "etcd": {
    "etcdserver": "3.3.17",
    "etcdcluster": "3.3.0"
  },
  "sensu_backend": "5.x.x#yyyyyyy"
}
```

### Context
6.0.0 https://github.com/sensu/sensu-go/pull/3965 and https://github.com/sensu/sensu-enterprise-go/pull/1203

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Updates etcd and Sensu version numbers in the examples in the version API doc for 6.0

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/2665

### CODE REVIEW

**Review by nikkictl — APPROVED** [review]

**Review by nikkictl — CHANGES_REQUESTED** [review]
Actually, I don't think the date shows up here anymore! I need to double check this.

File: content/sensu-go/6.0/api/version.md [hunk 1]
```diff
@@ -23,10 +23,10 @@ http://127.0.0.1:8080/version
 HTTP/1.1 200 OK
 {
   "etcd": {
-    "etcdserver": "3.3.17",
+    "etcdserver": "3.3.22",
     "etcdcluster": "3.3.0"
   },
-  "sensu_backend": "5.x.x#yyyyyyy"
+  "sensu_backend": "6.x.x#yyyyyyy"
```
Comment (nikkictl) [inline content/sensu-go/6.0/api/version.md]:
```suggestion
  "sensu_backend": "6.x.x"
```

**Comment by nikkictl** [comment]
Looks like the original issue was maybe edited? Here is the output!

```
$ curl https://127.0.0.1:8080/version
{"etcd":{"etcdserver":"3.3.22","etcdcluster":"3.0.0"},"sensu_backend":"6.0.0"}
```

File: content/sensu-go/6.0/api/version.md [hunk 2]
```diff
@@ -42,9 +42,9 @@ response parameters | Required: <ul><li>`etcd.etcdserver` (string). Etcd server
 output         | {{< code shell >}}
 {
   "etcd": {
-    "etcdserver": "3.3.17",
+    "etcdserver": "3.3.22",
     "etcdcluster": "3.3.0"
   },
-  "sensu_backend": "5.x.x#yyyyyyy"
+  "sensu_backend": "6.x.x#yyyyyyy"
```
Comment (hillaryfraley) [inline content/sensu-go/6.0/api/version.md:48]:
```suggestion
  "sensu_backend": "6.x.x"
```

File: content/sensu-go/6.0/api/version.md (hunk 1 above)
Comment (hillaryfraley) [reply content/sensu-go/6.0/api/version.md]: @nikkictl In the docs we say that the `yyyyyyy` thing is the SHA, not the date. Looks like it doesn't show up anymore either way though, is that right?

Comment (nikkictl) [reply content/sensu-go/6.0/api/version.md]: The SHA does not appear in the `/version` API (see https://github.com/sensu/sensu-docs/pull/2668#issuecomment-671488902). It only appears in the `sensu-backend version`, `sensu-agent version` and `sensuctl version` commands (in 6.0).

```
$ sensuctl version
sensuctl version 6.0.0+ee, enterprise edition, build dd1adf3892a8f408c2abd3655fb8f15ea1d77bd0, built 2020-08-10T16:46:13Z, built with go1.13.15
```

File: content/sensu-go/6.0/api/version.md (hunk 2 above)
Comment (nikkictl) [inline content/sensu-go/6.0/api/version.md]:
```suggestion
  "sensu_backend": "6.x.x"
```

**Comment by hillaryfraley** [comment]
Thank you, @nikkictl -- I tried to correct the description table too since the API doesn't return the SHA.

**Review by nikkictl — APPROVED** [review]

**Pull request merged**